src/cookies.njk

```diff
@@ -0,0 +1,82 @@
+---
+title: Cookies
+description: GOV.UK Design System puts small files (known as ‘cookies’) on your computer
+layout: layout-single-page.njk
+---
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set successNotificationBannerHtml %}
+  <p class="govuk-notification-banner__heading">You’ve set your cookie preferences.</p>
+{% endset %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="app-cookies-page" data-module="app-cookies-page">
+
+      {{ govukErrorSummary({
+        titleText: "There is a problem",
+        errorList: [
+          {
+            text: "Select yes to accept analytics cookies",
```

Comment:
Suggested text as per our error guidance.
I know we need to be careful when presenting choices with cookies, but think should be fine.
```suggestion
            text: "Select yes if you accept analytics cookies",
```